app/assets/stylesheets/application.scss

```diff
@@ -33,35 +35,40 @@ input, button, select, textarea {
   margin-right: 20px;
 }
 
-.home-header, .home-header2{
+.home_btn {
+  display: block;
+  border: 1px solid;
+  font-size: -webkit-xxx-large;
```

Comment:
<img width="216" alt="screen shot 2015-11-04 at 9 51 07 pm" src="https://cloud.githubusercontent.com/assets/13585275/10939663/31b08072-833e-11e5-876e-db1b5474ece9.png">

In inspect element, it had that choice.